README.md

```diff
@@ -48,6 +48,23 @@ For more information, you may [read the docs](https://github.com/intel/e2eAIOK).
 
 ## Installing
 
+### Install with Baremetal Environment
+
+ - To install e2eAIOK in baremetal environment, we recommend using [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/):
+
+    ```bash
+    conda create -n my_env 
+    conda activate my_env
+    pip install e2eAIOK
+    ```
+    To install latest nightly build, use `pip install --pre --upgrade e2eAIOK`
+
```

Comment:
so no option to install all?

Reply:
@zigzagcai , let's remove conda create and activate. I figured you're trying to make it looks like what bigdl did. But I found conda activate is not working in colab. So adding conda may not be recommended method.

Reply:
> so no option to install all?

`pip install e2eAIOK` this will install all components, including e2eaiok-sda, e2eaiok-denas and e2eaiok-modeladaptor

Reply:
pip install e2eAIOK will cover all components